docs/modpacks/creating_modpacks.md

```diff
@@ -52,6 +52,16 @@ When you have a working packwiz pack, you can use the [`packwiz mr export`] comm
 
 [`packwiz mr export`]: https://packwiz.infra.link/reference/commands/packwiz_modrinth_export/
 
+### mc-modpack-kit
+
+:::caution
+The project is quite new, so it may have some bugs.
+:::
+
+[`mc-modpack-kit`](https://github.com/jh-devv/mc-modpack-kit) is a GitHub Actions workflow template designed to automate the process of creating and releasing Minecraft modpacks. This template simplifies tasks such as version management, modpack building, and publishing to platforms like Modrinth and CurseForge. By using this template, you can streamline your modpack development and make it easier to share your creations with the community.
+
+You can export your modpack with mc-modpack-kit to Modrinth just by adding few GitHub secrets to your repo without even needing to touch GitHub Actions.
```

Comment:
This document isn't an ad for your project, it's a guide on how to create modpacks using these tools. Please rephrase this so that it reads less like an ad and more like instructions on how specifically to use this to create a Modrinth modpack?

Reply:
Yea sorry 😞 I'm pretty bad at writing stuff.
But I can do that!